content/en/apps/guides/training/training-cards/_index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Training Cards"
+linkTitle: "Training Cards"
+weight: 15
+description: >
+  Deploy in-app training cards for remote training.
+keyword: training
+relatedContent: >
+  apps/features/training
+  apps/examples/training
+  apps/examples/learning-care/
+---
+_Introduced in 4.2.0_
+
+[Training Cards]({{< relref "apps/features/training" >}}) enable remote training from within the CHT by showing a sequence of "cards" containing content provided by App Developers. Enketo forms are used to display the content, and App Developers can specify a start date, duration, and to which [user roles]({{< relref "apps/reference/app-settings/user-roles" >}}) the cards should be shown. Like [app forms]({{< relref "apps/reference/forms/app" >}}), forms used by training cards will automatically be downloaded to the user’s devices.
```

Comment:
Is it possible to include a short description of the kind of content that will be provided or the context of training cards. E.g that they explain changes in features or UI/UX

Reply:
Good idea! I've added a short description.